dsgrid/loggers.py

```diff
@@ -98,3 +99,22 @@ def check_log_file_size(filename, limit_mb=10, no_prompts=False):
         val = input(msg)
         if val == "" or val.lower() == "y":
             os.remove(filename)
+
+
+@contextmanager
+def disable_console_logging(name="dsgrid"):
+    logger = logging.getLogger(name)
+    console_level = None
+    try:
+        for handler in logger.handlers:
+            if handler.name == "console":
+                console_level = handler.level
+                handler.setLevel(logging.FATAL)
+                break
+        yield
+    finally:
+        for handler in logger.handlers:
+            if handler.name == "console":
+                assert console_level is not None
```

Comment:
nit: How do you know/expect console_level to not be None?

Reply:
Unless something crazy happened, it is the same for loop as the first one and `console_level` should have been set.